teps/0031-tekton-bundles-cli.md

```diff
@@ -66,3 +71,95 @@ Provide a way for users to...
 * Users may publish to any repository they have access to.
 
 * Only valid Tekton Bundles are produced.
+
+## Proposal
+
+Amend the Tekton CLI to provide the following commands (more detail later):
+
+```shell
+# REF = OCI reference, eg. docker.io/myworkspace/mybundle:1.0
+# BUNDLE_OBJECT = path to a file or directory or STDIN of a valid Tekton Task or Pipeline object to be included in the
+# bundle.
+
+# Generates a Tekton Bundle from a set of 1+ bundle objects. Like `docker`, tags the object with the specified reference
+# locally.
+tkn bundle build <REF> [BUNDLE_OBJECT...]
+
+# Like docker, publishing the locally referenced Tekton Bundle to a remote repository.
+tkn bundle push <REF>
+
+# Fetches a Tekton Bundle and prints its contents in a configurable format. If KIND is specified, will print only
+# objects of the specified kind (eg, Pipeline or Task). If KIND and NAME are specified, will retrieve a specific object.
+tkn bundle get <REF> [KIND] [NAME] --output=[FORMAT]
+
+# The following are amendments to existing commands.
+
+# Retrieves from an image the requested task(s) rather than from the cluster.
+tkn task list --image=<REF>
+tkn task get --image=<REF> [NAME]
```

Comment:
What is the output of this? Does it write yaml/json to stdout? Does it create a local bundle with only this task in?